Sources/BuildSystemIntegration/ExternalBuildSystemAdapter.swift

```diff
@@ -94,10 +94,34 @@ actor ExternalBuildSystemAdapter {
   private var lastRestart: Date?
 
   static package func projectRoot(for workspaceFolder: AbsolutePath, options: SourceKitLSPOptions) -> AbsolutePath? {
-    guard localFileSystem.isFile(workspaceFolder.appending(component: "buildServer.json")) else {
-      return nil
+    var buildServerConfigLocations = [String]()
```

Comment:
```suggestion
    var buildServerConfigLocations: [String] = []
```

is the preferred spelling.